week4:JSweek2/6.js

```diff
@@ -0,0 +1,7 @@
+let arr = [ ];
+console.log("the value of my array will be: undefined");
+console.log(arr[0]);
+arr = ['parrot', 'fish', 'dog'];
+console.log(arr[2]);
+arr.push('baby pig');
+console.log(arr[3]);
```

Comment:
You can also try log the entire array itself, eg `console.log(arr)`. What do you expect it to show?

Reply:
I dont even know why I referred to specific items.  :))))